src/dataservices/highlights.ts

```diff
@@ -82,4 +84,17 @@ export class HighlightsDataService {
 
     throw new NotFoundError();
   }
+
+  async deleteHighlightById(highlightId: string): Promise<string> {
+    // This will throw and error if it doesn't like you
+    const rowCount = await this.writeDb<HighlightEntity>('user_annotations')
+      .update({ status: 0 })
+      .where('user_id', this.userId)
+      .andWhere('annotation_id', highlightId);
```

Comment:
@kschelonka Question for yah.  This tests if the id doesn't exist in within this query. Are there more catastrophic failures we should be accounting for?